vcr/assets/assets/default.index.yaml

```diff
@@ -5,20 +5,63 @@ indices:
   - name: index_issuer
     parts:
       - iriPath: ["https://www.w3.org/2018/credentials#issuer"]
+  - name: index_subject_organization
+    parts:
+      - iriPath: ["https://www.w3.org/2018/credentials#credentialSubject"]
+      - iriPath:
+          [
+            "https://www.w3.org/2018/credentials#credentialSubject",
+            "http://schema.org/organization",
+            "http://schema.org/legalname",
+          ]
+        tokenizer: whitespaceOrExact
+        transformer: cologne
+      - iriPath:
+          [
+            "https://www.w3.org/2018/credentials#credentialSubject",
+            "http://schema.org/organization",
+            "http://schema.org/city",
+          ]
+        tokenizer: whitespaceOrExact
+        transformer: cologne
   - name: index_organization
```

Comment:
this indexes the same fields but in reverse order. This one isn't used?

Can't we sort the search fields for the available indexes?

Reply:
This one has an added `credentialSubject.id`. In the index this must be the first field since the other fields are queried with `not_nil`. The other index only has name and city. Leia uses the longest, best match index it can find. Query fields are sorted/matched to the index.